serpent-moltres/fuel-column/postprocessing.py

```diff
@@ -0,0 +1,34 @@
+import os
+import sys
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.cbook import get_sample_data
+import serpentTools as st
+import pathmagic
+from auxiliary import plot_serpent_axial_collapse
+
+
+if __name__ == "__main__":
+
+    A = 18/np.cos(np.pi/6)  # cm length of face of the hexagon
```

Comment:
file not pep8 compatible

Reply:
I don't understand. Running ```pycodestyle postprocessing.py```  doesn't throw any warnings/errors. I also searched for pep8 guidelines for main functions and I didn't find anything. Could you elaborate on your comment?

Reply:
I believe you were referring to the variable names. I changed them just in case.